app/Services/UserService.php

```diff
@@ -2,24 +2,64 @@
 
 namespace App\Services;
 
+use App\Models\User;
+use Firebase\JWT\JWT;
+use Firebase\JWT\Key;
+use App\Models\Invite;
+use App\Models\UserRole;
+use Firebase\JWT\ExpiredException;
+use App\Http\Resources\UserResource;
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Validation\UnauthorizedException;
 
 class UserService
 {
-    public function authenticateUser(array $validatedUserLogin): string
+    public function authenticateUser(array $validatedUserLogin): array
     {
         if (!Auth::attempt($validatedUserLogin)) {
             throw new UnauthorizedException("Invalid login credentials");
         }
 
         $user = Auth::user();
 
-        return $user->createToken('authToken')->plainTextToken;
+        $token = $user->createToken('authToken')->plainTextToken;
+
+        return [
+            'user' => UserResource::make($user),
```

Comment:
@sabinshrestha8 Make data transformation for data that is send to response from the controller